test_utils/browserMocks.js

```diff
@@ -0,0 +1,22 @@
+var localStorageMock = (function() {
+  var store = {};
+
+  return {
+    getItem: function(key) {
+      return store[key] || null;
+    },
+    setItem: function(key, value) {
+      store[key] = value;
```

Comment:
there was a `toString` here which caused some trouble, I removed that. Not sure if that changes heavily how a real browser would deal with it.. 

It was breaking when trying to persist `undefined` for the selected organisation for when there are no `orgs` yet in Happa.

Since there is no toString method on undefined.

But in a real browser, that worked fine, so I felt ok about changing this mock to make it work.